### PR TITLE
pythonPackages.tubeup: init at 0.0.19

### PR DESCRIPTION
**Pull request by marsam**
###### Motivation for this change
Add https://github.com/bibanon/tubeup

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by marsam** [comment]
@GrahamcOfBorg build python3Packages.tubeup

File: pkgs/development/python-modules/tubeup/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    inherit pname version;
+    sha256 = "6e3ebbf677a43018bfd71070919187bd57120010b0d708c0494c0f17bb72e84e";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py --replace "docopt==0.6.2" "docopt"
+  '';
+
+  propagatedBuildInputs = [ internetarchive docopt youtube-dl ];
+
```
Comment (jonringer) [inline pkgs/development/python-modules/tubeup/default.nix:26]: There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Comment (marsam) [reply pkgs/development/python-modules/tubeup/default.nix:26]: Thanks for reviewing! I think upstream tests are broken. I'll try to submit a patch later, and update this PR.

Comment (marsam) [reply pkgs/development/python-modules/tubeup/default.nix:26]: I opted to add `pythonImportsCheck` for now at least.

I created a PR upstream to fix the tests https://github.com/bibanon/tubeup/pull/127, but unfortunately I don't have much free time nowadays to finish it

**Pull request merged**